### PR TITLE
design-related proposals for core

### PR DESCRIPTION
**Pull request by andmironov**
Added design-related proposals for core

### CODE REVIEW

File: ideas/310-core-team-proposal.md [hunk 1]
```diff
@@ -227,6 +227,31 @@ We need to be able to see the same message history on multiple devices (desktop
...
+#### Design
+- Component-based UI; https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=0%3A1
+
+- New Profile https://www.figma.com/file/FwEv7oz7gW9KGS69RtKcbPuL/Identities?node-id=803%3A3933
+    - Make use of random names;
+    - Improve profile sharing;
+    - Easy account switching;
+    - New Public Chat, Private Chat, User and Dapp Profiles;
+- New Nav Structure -> New Tab bar https://framer.cloud/RAxDT
+- Blocked users;
```
Comment (mandrigin) [inline ideas/310-core-team-proposal.md]: I would move this to the Chat team responsibilities.

File: ideas/310-core-team-proposal.md [hunk 2]
```diff
@@ -227,6 +227,31 @@ We need to be able to see the same message history on multiple devices (desktop
...
+- Muted users;
```
Comment (mandrigin) [inline ideas/310-core-team-proposal.md]: I would move this to the Chat team responsibilities.

File: ideas/310-core-team-proposal.md [hunk 3]
```diff
@@ -227,6 +227,31 @@ We need to be able to see the same message history on multiple devices (desktop
...
+    - Improve settings structure;
+    - Notification settings;
+    - Privacy settings
+        - Who can in invite me to group chats;
+        - Who can send me messages;
+    - Security settings
+        - Introduce passcode lock;
+        - Add Face ID/Fingerprint support to replace passcode lock;
+    - Data Settings (Delete data/Export data)
+- Profile Info Requests in chat
```
Comment (mandrigin) [inline ideas/310-core-team-proposal.md]: I would move this to the Chat team responsibilities.

**Comment by oskarth** [comment]
Closing PR as out of date. A bunch of these problem belong to onboarding swarm.

**Pull request closed without merging**